### PR TITLE
Update observability_enable.adoc

### PR DESCRIPTION
**Pull request by dockerymick**
update for 2.3
https://github.com/open-cluster-management/backlog/issues/10553

@songleo I am unsure of what the comment should read for enableDownSampling

### CODE REVIEW

File: observability/observability_enable.adoc [hunk 1]
```diff
@@ -122,6 +122,7 @@ metadata:
   name: observability #Your customized name of MulticlusterObservability CR
 spec:
   availabilityConfig: High # Available values are High or Basic
+  enableDownSampling: true # The default value is true. This is not recommended as querying long-time ranges without non-downsampled data is not efficient and useful.
```
Comment (dockerymick) [inline observability/observability_enable.adoc]: @songleo the comment here reads the same as the comment in 2.2. Is a `true` value also not recommended?

Comment (songleo) [reply observability/observability_enable.adoc]: @dockerymick sorry for confusing, it should be `This is recommended as querying long-time ranges downsampled data is efficient and useful.`

Comment (dockerymick) [reply observability/observability_enable.adoc]: No worries! Thanks for clarifying. Making the change now and will continue to merge the PR

**Pull request merged**